analytic_engine/src/table/data.rs

```diff
@@ -175,7 +175,7 @@ impl Drop for TableData {
 
 #[inline]
 fn get_mutable_limit(opts: &TableOptions) -> u32 {
-    opts.write_buffer_size / 8 * 7
+    opts.write_buffer_size * 5 / 8
```

Comment:
This will lead to more small SST, which may lead to slow query response.

I think we can leave this unchanged, and optimize flush speed to avoid stall, also users can adjust write buffer to avoid stall.